code/styleproject.css

```diff
@@ -0,0 +1,51 @@
+body {font-family: Arial, Helvetica, sans-serif;
+  background-color: #e8fbf8;
+  text-align: center;
+  margin: 30px
+}
+
+img {width: 800px;}
+
+.rund{border-radius: 50%;}
+
```

Comment:
In your comment you mention that you want to round up your images, but I can't find the element/s that you applied this class to. Are you sure you are calling it by the right class so that border-radius is applied?